lib/repl.js

```diff
@@ -128,6 +128,10 @@ const {
 } = internalBinding('contextify');
 
 const history = require('internal/repl/history');
+let nextREPLResourceNumber = 1;
+function getREPLResourceName() {
```

Comment:
can you put in a comment explaining why this is needed

Reply:
```suggestion
// this prevents v8 code cache from getting confused and using a different
// cache from a resource of the same name
function getREPLResourceName() {
```